scripts/fill_virtualtopic_names.py

```
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+"""Rellena nombre y descripción en VirtualTopic existentes."""
+import sys
+import os
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from datetime import datetime
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)